src/components/MiniMap.vue

```diff
@@ -1,22 +1,39 @@
 <template>
-  <div class="mini-paper-container" @click="movePaper" :class="isOpen ? 'opened' : 'closed'">
-    <div ref="miniPaper" class="mini-paper" />
-  </div>
+  <b-button
+    class="btn btn-sm btn-secondary mini-map-btn ml-auto"
+    data-test="mini-map-btn"
+    @click="isOpen = !isOpen"
+    v-b-tooltip.hover
+    :title="isOpen ? $t('Hide Mini-Map') : $t('Show Mini-Map')"
+  >
+    <font-awesome-icon :icon="miniMapOpen ? minusIcon : mapIcon" />
+    <div class="mini-paper-container" @click="movePaper" :class="{ 'opened' : isOpen,  'closed': !isOpen, 'expanded' : expanded }">
```

Comment:
Should the mini-map close every time you click on it? What if someone is navigating a large diagram?

Reply:
Thanks for noticing, that was due to something I tried due to a styling issue. I found another solution instead and have fixed that.